src/forms/login-popup/data/reducers.js

```diff
@@ -53,10 +53,11 @@ export const loginSlice = createSlice({
         errorCode,
         email,
         value,
+        redirectUrl,
```

Comment:
was this data being returned by the API?

Reply:
Locally it is not being returned we need to test it for staging

Reply:
If it it not returned locally, i dont think it will be there on staging. Also this is onboarding component so there isnt specific staging env.